streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformer.java

```diff
@@ -87,7 +87,7 @@
 
```

Comment:
For my information, in `Transformer` to drop a key-value pair, `null` can be returned in `transform`. In ValueTransformer, is it applicable ?

Reply:
Not exactly. Returning `null` from `Transformer` causes the record not to be forwarded at all. Returning `null` from a `ValueTransformer` will result in forwarding a `(key, null)` pair, which my have different semantics downstream.

Reply:
Thanks for the information.